### DISCUSSION TITLE
SMSAPI - "Rodzaj wiadomości"

### DISCUSSION THREAD

**Discussion by mateusz-iwanski** [YetiForceCompany/YetiForceCRM#16025] [How to] [answered]
Czy "Rodzaj wiadomości" to "Pole nadawcy" dla bramki? Gdzie mogę dodać nowy rodzaj lub edytować domyślne "Eco"?
![Zrzut ekranu 2022-05-03 202401](https://user-images.githubusercontent.com/2405443/166518083-6d8e9db5-f9d3-4fc0-9a21-ddec3b1da925.png)

**Comment by rskrzypczak** [comment, marked as answer]
Tak, aczkolwiek widać, że używasz starej wersji systemu, gdyż już od kilku wersji pole nadawcy nie jest słownikiem.
Z tego też powodu komunikacja z SMSAPI może Ci nie działać.

**Comment by mateusz-iwanski** [comment]
Dzięki, racja, po aktualizacji działa prawidłowo.
